lib/transports/rest2.js

```diff
@@ -250,6 +254,36 @@ class RESTv2 {
     return new ModelClass(data)
   }
 
+  /**
+   * @param {Arr} data order matters
```

Comment:
`{Arr}` should be `{Array}` or `{[]}`; if it's an array of arrays, use `{Array[]}` to clarify

Reply:
Cris i dont understand this, company is a string, example : 'bitfinex'